src/api/handlers/appointments/sendReminders.js

```diff
@@ -65,9 +72,12 @@ export const sendReminders = async () => {
       "patientPhoneNumber",
       "practitionerAddress",
       "specialNotes",
-      "description"
+      "description",
+      "lastReminderSentAt"
     )
     .where("isDeleted", false)
+    .where("appointmentIsConfirmed", null) // whereNot ("...", false) does not work
```

Comment:
These two checks on `appointmentIsConfirmed` need to be grouped together similar to what I did below with `lastReminderSentAt`. Otherwise, this will select every appointment where `appointmentIsConfirmed` is `null`.

```js
.andWhere((db) => {
db.where("appointmentIsConfirmed", true).orWhereNull("appointmentIsConfirmed");
})
```

Reply:
Took another stab at it. Lemme know if you think that will work!

Reply:
Nope, this change is functionally equivalent to what you had before. You need to make sure that the generated SQL is putting both `appointmentIsConfirmed` checks into parenthesis.


Currently:

```SQL
SELECT <omitted>
FROM   `appointments`
WHERE  `isdeleted` = false
       AND `appointmentisconfirmed` IS NULL
        OR `appointmentisconfirmed`
           AND `appointmenttime` >= '2020-11-07 04:48:41'
           AND `appointmenttime` < '2020-11-08 04:48:41'
           AND ( `lastremindersentat` <= '2020-11-06 04:48:41'
                  OR `lastremindersentat` IS NULL )  
```


Needed:

```SQL
SELECT <omitted>
FROM   `appointments`
WHERE  `isdeleted` = false
       AND ( `appointmentisconfirmed` = true
              OR `appointmentisconfirmed` IS NULL )
       AND `appointmenttime` >= '2020-11-07 04:46:22'
       AND `appointmenttime` < '2020-11-08 04:46:22'
       AND ( `lastremindersentat` <= '2020-11-06 04:46:22'
              OR `lastremindersentat` IS NULL )  
```

You can do this by putting both into an `andWhere` like in my previous code snippet.

Reply:
Alright. I goofed on the copy pasted and removed the "()" from my raw so it was messing it up. I converted it to Knex style like you mentioned and it outputs this now:
```
select 
	`appointmentId` as `id`, 
	`patientName`, 
	`language`, 
	`appointmentTime`, 
	`patientPhoneNumber`, 
	`practitionerAddress`, 
	`specialNotes`, 
	`description`, 
	`lastReminderSentAt` 
from 
	`sanctuary_dev`.`appointments` 
where 
	`isDeleted` = false 
	and	(`appointmentIsConfirmed` = true or `appointmentIsConfirmed` is null) 
	and `appointmentTime` >= 'dateTime' 
	and `appointmentTime` < 'dateTime' 
	and (
			`lastReminderSentAt` <= 'dateTime' or 
			`lastReminderSentAt` is null
	)
```